requirements.txt

```diff
@@ -43,6 +43,9 @@ cffi==1.11.2
 # aws commandline tools for running on Docker remotely.
 awscli>=1.11.91
 
+# Accessing files from S3 directly.
+boto3
```

Comment:
you'll need to add this requirement to `setup.py` as well

Reply:
Done, also added `moto` to requirements.txt and setup.py